src/parse/ris_live.go

```diff
@@ -77,7 +78,7 @@ type RisLiveMessage struct {
 }
 
 // connects to ris live, starts go routine receiverHandler, manages connection and subscription
-func ParseRisLiveData(msgChannel chan common.BGPMessage) error {
+func ParseRisLiveData(msgChannel chan common.BGPMessage, configPrefixes string) error {
```

Comment:
this is okay *but* you could also pass `config *config.Configuration` (I think) and then refer to the parts that you need. Means that if there are more parts of the config that you need to refer to, then this function signature doesn't need to change. (e.g., socketUrl is highly unlikely to change but, if it does, it could be updated in the config more easily than rebuilding!)